app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+def copy_file(command: str) -> None:
+    standard_copy_command = "cp"
+    copy_command_words_total = 3
+    copy_command_error_message = "Bad Command"
+
+    normalized_command = command.split(" ")
+
+    if len(normalized_command) != copy_command_words_total:
+        raise Exception(copy_command_error_message)
+
+    copy_command, file_to_copy_path, destination_path = normalized_command
+
+    if copy_command != standard_copy_command:
+        raise Exception(copy_command_error_message)
+
+    if file_to_copy_path == destination_path:
+        return
```

Comment:
try not to use blank return here, make a backward condition

Reply:
I do not think there is a need to do this as it increases nesting